lib/errorUtils.js

```diff
@@ -0,0 +1,10 @@
+function reshapeExceptionError(error) {
+    return {
+        message: error.message,
+        code: error.code,
```

Comment:
From the docs, some attributes that can be useful to capture are also `stack` and `name`.